jenkins/codecov_response_metrics.py

```diff
@@ -43,7 +43,8 @@
     'edx/edx-notes-api',
     'edx/edx-app-android',
     'edx/edx-video-worker',
-    'edx/edx-platform'
+    'edx/edx-platform',
+    'edx/bok-choy'
```

Comment:
nit: I think it would be a little easier to scan this list if it was alphabetized, and adding a trailing comma after the last entry would simplify future diffs